diff --git a/CMakeLists.txt b/CMakeLists.txt
index b5ffafa..a16ad19 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,21 +30,21 @@ include_directories(
     Common/Types/C/ASNCommon
     Common/Types/C Common/Converters
     Common/Types/Cpp
-    External/install/include
-    External/install/include/boost
-    External/install/include/eigen3
-    External/install/include/flann
-    External/install/include/libqhull
-    External/install/include/libqhullcpp
-    External/install/include/libqhull_r
-    External/install/include/opencv
-    External/install/include/opencv2
-    External/install/include/pcl-1.8
-    External/install/include/vtk-8.1
-    External/install/include/yaml-cpp
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/boost
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/eigen3
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/flann
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/libqhull
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/libqhullcpp
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/libqhull_r
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/opencv
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/opencv2
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/pcl-1.8
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/vtk-8.1
+    $ENV{AUTOPROJ_CURRENT_ROOT}/install/include/yaml-cpp
 )
 link_directories(
-	 External/install/lib
+	 $ENV{AUTOPROJ_CURRENT_ROOT}/install/lib
 )
 
 add_subdirectory(Common)
