diff --git a/CC/CMakeLists.txt b/CC/CMakeLists.txt
index 211e6d94..bc41be27 100644
--- a/CC/CMakeLists.txt
+++ b/CC/CMakeLists.txt
@@ -75,6 +75,7 @@ if ( COMPILE_CC_CORE_LIB_SHARED )
 		endforeach()
 	else()
 		install_shared( cloudcompare_core ${CMAKE_INSTALL_PREFIX}/lib 0 ) #default destination: /usr/local/lib
+		install(FILES ${header_list} DESTINATION "${CMAKE_INSTALL_PREFIX}/include/cloudcompare-core") #default destination: /usr/local/include/cloudcompare-core
 	endif()
 endif()
 
