diff --git a/CC/CMakeLists.txt b/CC/CMakeLists.txt
index d9aaa5b9..211e6d94 100644
--- a/CC/CMakeLists.txt
+++ b/CC/CMakeLists.txt
@@ -74,7 +74,7 @@ if ( COMPILE_CC_CORE_LIB_SHARED )
 			install_shared( cloudcompare_core ${dest} 1 )
 		endforeach()
 	else()
-		install_shared( cloudcompare_core ${CMAKE_INSTALL_LIBDIR}/cloudcompare 0 ) #default destination: /usr/lib/cloudcompare
+		install_shared( cloudcompare_core ${CMAKE_INSTALL_PREFIX}/lib 0 ) #default destination: /usr/local/lib
 	endif()
 endif()
 
