diff --git a/CMakeLists.txt b/CMakeLists.txt
index db9b1ea..810e4fe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -85,7 +85,7 @@ endif()
 if(USE_BUNDLED_DEPENDENCIES)
 
   set(CMAKE_MODULE_PATH
-    "${PROJECT_SOURCE_DIR}/${BUNDLED_DEPENDENCIES_PREFIX}/share/cmake-3.11.4/Modules")
+    "${BUNDLED_DEPENDENCIES_PREFIX}/share/cmake-3.11.0-rc4/Modules")
 
 else()
 
@@ -144,7 +144,7 @@ endmacro()
 # CMake will look for the headers and the requested compiled libraries, as well
 # as the compiled libraries they depend on.
 
-find_package(Boost 1.66.0 REQUIRED)
+find_package(Boost REQUIRED)
 
 if(Boost_FOUND)
 
@@ -222,6 +222,7 @@ endif()
 # TODO: remove, use imported target Eigen3::Eigen instead
 include_directories(
   SYSTEM "${_DEPENDENCIES_PREFIX}/include/eigen3"
+  "/usr/include/eigen3"
 )
 
 ## Look for Ceres --------------------------------------------------------------
@@ -278,16 +279,17 @@ endif()
 
 include_directories(
   SYSTEM "${_DEPENDENCIES_PREFIX}/include"
+  "/usr/include"
 )
 
 ## Look for VTK, PCL, OpenCV ---------------------------------------------------
 
 # TODO: rework this section
 
-find_package(VTK 8.1.0 REQUIRED NAMES VTK VTK-8.1 CONFIGS VTKConfig.cmake)
+find_package(VTK REQUIRED NAMES VTK VTK-8.1 CONFIGS VTKConfig.cmake)
 find_package_mode(VTK)
 
-find_package(PCL 1.8.1 REQUIRED NAMES PCL PCL-1.8 CONFIGS PCLConfig.cmake)
+find_package(PCL REQUIRED NAMES PCL PCL-1.8 CONFIGS PCLConfig.cmake)
 find_package_mode(PCL)
 add_definitions(${PCL_DEFINITIONS})
 include_directories(SYSTEM ${PCL_INCLUDE_DIRS})
@@ -298,9 +300,11 @@ message(STATUS "Found PCL headers, and the headers of its dependencies, in: ${PC
 message(STATUS "Found PCL libraries in: ${PCL_LIBRARY_DIRS}")
 message(STATUS "All requested PCL libraries, and their dependencies: ${PCL_LIBRARIES}")
 
-find_package(OpenCV 3.4.0 REQUIRED)
+find_package(OpenCV REQUIRED)
 find_package_mode(OpenCV)
 
+link_directories("/usr/lib")
+
 ## Look for Edres-Wrapper (optional dependency) --------------------------------
 
 find_package(Edres-Wrapper QUIET)
