diff --git a/CMakeLists.txt b/CMakeLists.txt
index fed9188..59024a5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -85,7 +85,7 @@ endif()
 if(USE_BUNDLED_DEPENDENCIES)
 
   set(CMAKE_MODULE_PATH
-    "${PROJECT_SOURCE_DIR}/${BUNDLED_DEPENDENCIES_PREFIX}/share/cmake-3.11.4/Modules")
+    "${BUNDLED_DEPENDENCIES_PREFIX}/share/cmake-3.11.0-rc4/Modules")
 
 else()
 
@@ -144,7 +144,7 @@ endmacro()
 # CMake will look for the headers and the requested compiled libraries, as well
 # as the compiled libraries they depend on.
 
-find_package(Boost 1.66.0 REQUIRED)
+find_package(Boost REQUIRED)
 
 if(Boost_FOUND)
 
@@ -225,6 +225,7 @@ endif()
 # TODO: remove, use imported target Eigen3::Eigen instead
 include_directories(
   SYSTEM "${_DEPENDENCIES_PREFIX}/include/eigen3"
+  "/usr/include/eigen3"
 )
 
 ## Look for Ceres --------------------------------------------------------------
@@ -281,19 +282,20 @@ endif()
 
 include_directories(
   SYSTEM "${_DEPENDENCIES_PREFIX}/include"
+  "/usr/include"
 )
 
 ## Look for VTK, PCL, OpenCV ---------------------------------------------------
 
 # TODO: rework this section
 
-find_package(VTK 8.1.0 REQUIRED NAMES VTK VTK-8.1 CONFIGS VTKConfig.cmake)
+find_package(VTK REQUIRED NAMES VTK VTK-8.1 CONFIGS VTKConfig.cmake)
 find_package_mode(VTK)
 
-find_package(PCL 1.8.1 REQUIRED NAMES PCL PCL-1.8 CONFIGS PCLConfig.cmake)
+find_package(PCL REQUIRED NAMES PCL PCL-1.8 CONFIGS PCLConfig.cmake)
 find_package_mode(PCL)
 
-find_package(OpenCV 3.4.0 REQUIRED)
+find_package(OpenCV REQUIRED)
 find_package_mode(OpenCV)
 
 include_directories(
@@ -329,6 +331,7 @@ endif()
 
 link_directories(
   "${_DEPENDENCIES_PREFIX}/lib"
+  "/usr/lib"
 )
 
 ## CDFF ------------------------------------------------------------------------
