diff --git a/CMakeLists.txt b/CMakeLists.txt
index 84f0e8c..c3c4f73 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,7 +84,7 @@ endif()
 if(USE_BUNDLED_DEPENDENCIES)
 
   set(CMAKE_MODULE_PATH
-    "${PROJECT_SOURCE_DIR}/${BUNDLED_DEPENDENCIES_PREFIX}/share/cmake-3.11.0-rc4/Modules")
+    "${BUNDLED_DEPENDENCIES_PREFIX}/share/cmake-3.11.0-rc4/Modules")
 
 else()
 
@@ -143,7 +143,7 @@ endmacro()
 # CMake will look for the headers and the requested compiled libraries, as well
 # as the compiled libraries they depend on.
 
-find_package(Boost 1.66.0 REQUIRED COMPONENTS system)
+find_package(Boost REQUIRED COMPONENTS system)
 
 if(Boost_FOUND)
 
@@ -220,6 +220,7 @@ endif()
 # TODO: remove, use imported target Eigen3::Eigen instead
 include_directories(
   SYSTEM "${_DEPENDENCIES_PREFIX}/include/eigen3"
+  "/usr/include/eigen3"
 )
 
 ## Don't look for FLANN, QHull, yaml-cpp ---------------------------------------
@@ -233,19 +234,20 @@ include_directories(
 
 include_directories(
   SYSTEM "${_DEPENDENCIES_PREFIX}/include"
+  "/usr/include"
 )
 
 ## Look for VTK, PCL, OpenCV ---------------------------------------------------
 
 # TODO: rework this section
 
-find_package(VTK 8.1.0 REQUIRED NAMES VTK VTK-8.1 CONFIGS VTKConfig.cmake)
+find_package(VTK REQUIRED NAMES VTK VTK-8.1 CONFIGS VTKConfig.cmake)
 find_package_mode(VTK)
 
-find_package(PCL 1.8.1 REQUIRED NAMES PCL PCL-1.8 CONFIGS PCLConfig.cmake)
+find_package(PCL REQUIRED NAMES PCL PCL-1.8 CONFIGS PCLConfig.cmake)
 find_package_mode(PCL)
 
-find_package(OpenCV 3.4.0 REQUIRED)
+find_package(OpenCV REQUIRED)
 find_package_mode(OpenCV)
 
 include_directories(
@@ -262,6 +264,7 @@ include_directories(
 
 link_directories(
   "${_DEPENDENCIES_PREFIX}/lib"
+  "/usr/lib"
 )
 
 ## CDFF ------------------------------------------------------------------------
